src/main/webapp/components/dashboard/dashboard.html

```diff
@@ -0,0 +1,30 @@
+<div class="container-fluid">
+    <div class="row content">
+        <div class="col-sm-3 sidenav">
+            <h4>Shared Timesheet</h4>
+            <ul class="nav nav-pills nav-stacked">
+                <li ng-click="dashboardController.menuSelection('employee') "
+                    ng-class="{ 'active' : dashboardController.role == 'employee' }"><a href="#employee">Employee</a></li>
+
+                <li ng-click="dashboardController.menuSelection('internalManager') "
+                    ng-class="{ 'active' : dashboardController.role == 'internalManager' }"><a href="#internalManager">Internal
```

Comment:
508  Accessibility compliance.  ng-click and an 'a' tag. This is a case of "nested interactive elements" which will cause issues. 
https://accessibilityinsights.io/info-examples/web/nested-interactive/